Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -1331,7 +1331,7 @@
   icon: { sprite: /Textures/Clothing/OuterClothing/Hardsuits/syndieelite.rsi, state: icon }
   productEntity: ClothingBackpackDuffelSyndicateEliteHardsuitBundle
   cost:
-    Telecrystal: 12
+    Telecrystal: 10 #dv change- makes it the same cost as it was beofre
```

```suggestion
    Telecrystal: 10 #DeltaV - makes it the same cost as it was before
```
Spelling error. It's also good to spell out the name so people know where exactly it came from